smoke-tests/src/test/java/com/solarwinds/agents/Agent.java

```diff
@@ -40,7 +40,9 @@ public Agent(String name, String description) {
   }
 
   public Agent(String name, String description, String url) {
-    this(name, description, url, Collections.singletonList("-Dio.opentelemetry.context.enableStrictContext=true"));
+    this(name, description, url, Arrays.asList("-Dio.opentelemetry.context.enableStrictContext=true",
+            "-Dotel.java.experimental.span-stacktrace.min.duration=0ms",
+            "-Dsw.apm.span.stack.trace.filters=thread.id,os.description,http.request.method"));
```

Comment:
Nit and subjective: there are a lot of dots in this property `sw.apm.span.stack.trace.filters` and seems upstream (or at least semconv) uses single `stacktrace` rather than two-word version. Consider `sw.apm.span.stacktrace.filters` instead? but feel free to say nah ;)